.github/workflows/tests.yml

```diff
@@ -46,8 +46,7 @@ jobs:
           echo "::add-path::/home/linuxbrew/.linuxbrew/bin:/usr/bin:/bin"
         fi
 
-    - name: Test uninstaling GitHub Actions macOS Homebrew
-      if: runner.os == 'macOS'
```

Comment:
Leave this line. It's for uninstalling Homebrew which is preinstalled on macOS but not on Ubuntu.

Reply:
Noted. Will revert in a subsequent commit (just noticed a speling error in the original).

Reply:
My bad! If you could fix those it'd be great ❤️